packages/esm-messages-app/src/constants.ts

```diff
@@ -0,0 +1,47 @@
+export const datePickerPlaceHolder = 'dd/mm/yyyy';
```

Comment:
So, I don't think it sustainable to keep all of this hard-coded like this. How would we localize it? How would we support non-Gregorian calendars?

Reply:
adding  support for localization and non gregorian calenders in a  follow up PR